conf/openrefine.l4j.ini

```diff
@@ -8,6 +8,16 @@
 
 # Use system defined HTTP proxies
 -Djava.net.useSystemProxies=true
+# You should probably defined the properties below, if they are not already 
+# defined in the environment variable JAVA_TOOL_OPTIONS.
+# Refer to the Java documentation (for example, at
+# https://docs.oracle.com/javase/8/docs/api/java/net/doc-files/net-properties.html
+# for the precise meaning of each option
```

Comment:
```suggestion
# for the precise meaning of each option)
```

Reply:
Thanks for the review,  the last commit should include the typos.